docs/guides/qiskit-transpiler-service.mdx

```diff
@@ -94,6 +94,16 @@ The following examples demonstrate how to transpile circuits using the Qiskit Tr
     transpiled_circuit = cloud_transpiler_service.run(circuit)
     ```
 
+## Limits of the Qiskit Transpiler Service
+
+The service has some limits enabled, here we list the most relevant ones:
+
+- The Qiskit Transpiler Service allows to transpile up to 1 million two-qubit gates in either `ai` mode `ai="true"`, `ai="false"` or `ai="auto"`.
+- The maximum time allowed to run a transpilation process is 30 mins.
+- The maximum time to retrieve a transpilation result from the service after the transpilation process ends is 20 mins. After 20 mins your job will be removed.
+- The maximum time a set of circuits can live in the internal queues while waiting to be transpiled is 120 mins. After that time, if the job has not been transpiled, it will be discarded.
+- The maximum number of qubits is not really fixed. Though, the service has been only tested up to 900+ qubits.
```

Comment:
```suggestion
- The maximum time a set of circuits can live in the internal queue while waiting to be transpiled is 120 minutes. After that time, if the job has not been transpiled, it will be discarded.
- The maximum number of qubits has not been determined. The service has been tested up to 900+ qubits.
```